src/browser/modules/Stream/SchemaFrame.jsx

```diff
@@ -32,7 +35,7 @@ import {
   StyledBodyTr,
   StyledTd
 } from 'browser-components/DataTables'
-import { Directives } from 'browser-components/Directives'
+import Directives from 'browser-components/Directives'
```

Comment:
Not sure why, but after I connected `SchemaFrame` to redux, the `onItemClick` function from `Directives` wouldn't work unless I imported the connected version.